starfish/core/morphology/Filter/reduce.py

```diff
@@ -0,0 +1,95 @@
+from typing import Callable, Optional, Tuple, Union
+
+import numpy as np
+
+from starfish.core.morphology.binary_mask import BinaryMaskCollection
+from starfish.core.types import FunctionSource, FunctionSourceBundle
+from ._base import FilterAlgorithm
+
+
+class Reduce(FilterAlgorithm):
+    """
+    Reduce the masks from one ``BinaryMaskCollection`` using by applying a specified function
```

Comment:
```suggestion
    Reduce the masks to one ``BinaryMaskCollection`` by applying a specified function
```

Reply:
I have attempted to clarify the language here.  It is indeed taking the masks from a single `BinaryMaskCollection` but I can see how the wording is confusing.